src/pages/ContentfulPage.vue

```diff
@@ -142,7 +146,7 @@ const getComponentFromType = type => {
 		case 'homepageBottomCTA':
 			return HomepageBottomCTA;
 		case 'homepageHowItWorks':
-			return HomepageHowItWorks;
+			return MultiStepCentered;
```

Comment:
We'll need to preserve the previous switch here and add a new case switch for `RichTextItemsCentered` return `richTextItemsCentered`.